### DISCUSSION TITLE
@next/bundle-analyzer Where do I write other configurations?

### DISCUSSION THREAD

**Discussion by parlay96** [vercel/next.js#30508] [Help]
### Describe the feature you'd like to request

![Uploading image.png…]()
Where do I write other configurations?

{
  distDir: 'dist',
  experimental: { scss: true },
  useFileSystemPublicRoutes: false,
  compress: true,
  eslint: {
    ignoreDuringBuilds: false,
  },
  reactStrictMode: true, 
}

### Describe the solution you'd like

![Uploading image.png…]()
Where do I write other configurations?

### Describe alternatives you've considered

![image](https://user-images.githubusercontent.com/32978177/139182736-6cbd47a5-23aa-4d7d-bbbb-75e90d055d41.png)
Where do I write other configurations?

**Comment by ijjk** [comment]
Hi, the any additional configuration you would normally export directly goes inside of `withBundleAnalyzer({ /* config here */)` example:

```js
// before
module.exports = { reactStrictMode: true }
```

```js
// after
const withBundleAnalyzer = require('@next/bundle-analyzer')({
  enabled: process.env.ANALYZE === 'true',
})
module.exports = withBundleAnalzyer({ reactStrictMode: true })
```

**Reply by parlay96** [reply]
thank you
